aws/resource_aws_network_acl_rule_test.go

```diff
@@ -15,20 +15,34 @@ import (
 )
 
 func TestAccAWSNetworkAclRule_basic(t *testing.T) {
-	var networkAcl ec2.NetworkAcl
-
+	rnBaz := "aws_network_acl_rule.baz"
+	rnQux := "aws_network_acl_rule.qux"
+	rnWibble := "aws_network_acl_rule.wibble"
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckAWSNetworkAclRuleDestroy,
 		Steps: []resource.TestStep{
 			{
 				Config: testAccAWSNetworkAclRuleBasicConfig,
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckAWSNetworkAclRuleExists("aws_network_acl_rule.baz", &networkAcl),
-					testAccCheckAWSNetworkAclRuleExists("aws_network_acl_rule.qux", &networkAcl),
-					testAccCheckAWSNetworkAclRuleExists("aws_network_acl_rule.wibble", &networkAcl),
-				),
```

Comment:
For consistency with the rest of the provider codebase, let's keep these `Check`s (here and in the below tests) in the testing, even with import testing. We can discuss their potential removal separately. 👍

Reply:
ah will do!